filebeat/module/iis/w3svc/_meta/fields.yml

```diff
@@ -0,0 +1,115 @@
+- name: access
+  type: group
+  description: >
+    Contains fields for the Apache2 HTTPD access logs.
```

Comment:
The description contained in this documentation appears to incorrectly refers to "Apache2 HTTPD access logs". I think this description should read:

> Contains fields for IIS access logs.

Furthermore, the fields described in this file don't appear to match those configured in ` filebeat/module/iis/w3svc/ingest/pipeline.json`. It appears that these fields are a copy and paste of the Apache2 fields. Could this file be updated to include the relevant fields? See my [own fork for an initial list of the required fields](https://github.com/cgwrench/beats/blob/iis-filebeat-module/filebeat/module/iis/access/_meta/fields.yml).